lib/frame.rb

```diff
@@ -0,0 +1,44 @@
+class Frame 
+
+attr_reader :knocked_pins, :frame, :frame_id, :frame_score, :strike, :spare
+
+  def initialize
+    @frame_id = 0
+    @frame = {}
+  end
+
+  def first_roll(knocked_pins1)
+    @knocked_pins = knocked_pins1
+    if knocked_pins == 10
+       @frame = {frame_id: @frame_id += 1, first_roll: 'strike' }
+    else
+      @frame = {frame_id: @frame_id += 1, first_roll: knocked_pins1 }
+    end
```

Comment:
What is @frame_id doing? 
As this is common to both strings, you could pull it out of the conditional - avoids duplication.